ivadomed/transforms.py

```diff
@@ -447,11 +470,13 @@ def __call__(self, sample):
 
         if self.labeled:
             gt_data = sample['gt']
-            gt_vol = np.zeros(gt_data.shape)
-            for idx, gt in enumerate(gt_data):
-                pil_img = Image.fromarray(gt, mode='F')
-                gt_vol[idx, :, :] = np.array(self.sample_augment(pil_img, params))
-            ret_gt = gt_vol.astype('float32')
+            ret_gt = []
+            for labels in gt_data:
+                gt_vol = np.zeros(labels.shape)
+                for idx, gt in enumerate(labels):
+                    pil_img = Image.fromarray(gt, mode='F')
+                    gt_vol[idx, :, :] = np.array(self.sample_augment(pil_img, params))
+                ret_gt.append(gt_vol.astype('float32'))
```

Comment:
Do you need `float32`?

Reply:
I thought that Anthime modified the loader / transforms so that the gt are not float32. Good to know!
Yep, could do this during medicaltorch-ivadomed fusion :-)